setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-install_reqs = ["Django>=1.4", "sixpack-client==1.0.0"]
+install_reqs = ["Django>=1.4", "sixpack-client==1.1.1"]
```

Comment:
Good point....

I ran the migration in accounting with `pip install -e ../sandbox/djang-sixpack` and it picked up and migrated okay. Should we use 1.5 as the base?